bandit/cli/main.py

```diff
@@ -215,7 +215,7 @@ def main():
     )
     parser.add_argument(
         '-o', '--output', dest='output_file', action='store', nargs='?',
-        type=argparse.FileType('w'), default=sys.stdout,
+        type=argparse.FileType('w', encoding='utf-8'), default=sys.stdout,
```

Comment:
The encoding parameter wasn't added until Py3.4, but we still want to support Py2.7 at this point.

https://docs.python.org/3/library/argparse.html#filetype-objects